manifests/t/TeamSpeakSystems/TeamSpeakClient/Beta/5.0.0/TeamSpeakSystems.TeamSpeakClient.Beta.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TeamSpeakSystems.TeamSpeakClient.Beta
+PackageVersion: 5.0.0
+PackageLocale: en-US
+Publisher: TeamSpeak
+PackageName: TeamSpeak

```

```suggestion
PackageName: TeamSpeak Beta
```
Necessary to not match control panel here to not break current installations of TeamSpeak 3